src/views/shared/TextBox.js

```diff
@@ -23,85 +23,17 @@ define([
 ],
 function ($, Handlebars, BrowserFeatures, TextBox) {
 
-  function hasTitleAndText(options) {
-    var title = options.title,
-        text = options.text;
-
-    if (title && text && title !== text) {
-      return true;
-    }
-    return false;
-  }
-
-  // options may be a string or an object.
-  function createQtipContent(options) {
-    if (hasTitleAndText(options)) {
-      return options;
-    }
-    return {text: options.text || options};
-  }
-
   return TextBox.extend({
-
-    template: Handlebars.compile('\
-      {{#if params}}\
-        {{#if params.innerTooltip}}\
-          <span class="input-tooltip icon form-help-16"></span>\
-        {{/if}}\
-        {{#if params.icon}}\
-          <span class="icon input-icon {{params.icon}}"></span>\
-        {{/if}}\
-      {{/if}}\
-      <input type="{{type}}" placeholder="{{placeholder}}" aria-label="{{placeholder}}"\
-        name="{{name}}" id="{{inputId}}" value="{{value}}" autocomplete="off"/>\
-    '),
-
     postRender: function () {
-      var params = this.options.params,
-          content;
-
       if (this.options.type === 'number') {
         var input = this.$('input');
         input.attr({
           pattern: '[0-9]*',
           inputmode: 'numeric'
         });
       }
-
-      if (params && params.innerTooltip) {
```

Comment:
This logic is now moved into courage Textbox.js component https://github.com/okta/courage/blob/master/src/views/forms/inputs/TextBox.js#L106